autobisect-js/autoBisect.py

```diff
@@ -687,8 +687,8 @@ def getHgwebMozillaOrg(branchName):
 
 
 def getIdFromTboxUrl(url):
-    """Return numeric ID from treeherder at https://archive.mozilla.org/pub/mozilla.org/firefox/treeherder-builds/ ."""
-    return filter(None, url.split('/'))[-1]
+    """Return numeric ID from treeherder at https://archive.mozilla.org/pub/firefox/tinderbox-builds/ ."""
+    return [i for i in url.split("/") if i is not None][-2]  # There should be a trailing "/"
```

Comment:
Ah ok, so that was the reason for `[-1]` to `[-2]`. `if i` would be doing the same as `filter(None,...)` so you could keep `[-1]` and it wouldn't matter if the trailing slash is there or not.